pandas/tests/test_graphics.py

```diff
@@ -420,22 +420,20 @@ def _check_box_return_type(self, returned, return_type, expected_keys=None,
         from matplotlib.axes import Axes
         types = {'dict': dict, 'axes': Axes, 'both': tuple}
         if expected_keys is None:
-            # should be fixed when the returning default is changed
             if return_type is None:
```

Comment:
this should be an error if ``return_type`` is not valid (e.g. ``axes``)? ``None`` is no longer acceptable.